modules/multi-architecture-upgrade-mirrors.adoc

```diff
@@ -5,9 +5,9 @@
 :_content-type: PROCEDURE
 [id="multi-architecture-upgrade-mirrors_{context}"]
 
-= Upgrading your multi-architecture cluster
+= Upgrading your multi-architecture compute machines 
 
-You must perform an explicit upgrade command to upgrade your existing cluster to a multi-architecture cluster.
+You must perform an explicit upgrade command to upgrade your existing cluster to a cluster with multi-architecture compute machines.
 
```

Comment:
should be - "You must perform an explicit upgrade command to upgrade your existing cluster's payload to a payload which supports multi-architecture compute machines"